### PR TITLE
MeanpT fluctuations of charged particles

### PR DESCRIPTION
**Pull request by SwatiSaha-1997**

### CODE REVIEW

**Review by victor-gonzalez — CHANGES_REQUESTED** [review]
Please, implement my comments regarding renaming and commenting the columns/table

For the other just consider it if you want to simplify your code

File: PWGCF/EbyEFluctuations/Tasks/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+#
+# This software is distributed under the terms of the GNU General Public
+# License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+#
+# In applying this license CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+o2physics_add_dpl_workflow(task-ptcorrelation
+                    SOURCES TaskPtCorrelation.cxx
```
[the remark this answers is not in the document]
Comment (SwatiSaha-1997) [reply PWGCF/EbyEFluctuations/Tasks/CMakeLists.txt]: Done.

File: PWGCF/EbyEFluctuations/Tasks/TaskPtCorrelation.cxx [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+namespace ptQn
+{
+DECLARE_SOA_COLUMN(Q1, q1, float);
+DECLARE_SOA_COLUMN(Q2, q2, float);
+DECLARE_SOA_COLUMN(Q3, q3, float);
+DECLARE_SOA_COLUMN(Q4, q4, float);
+DECLARE_SOA_COLUMN(N_ch, n_ch, float);
+DECLARE_SOA_COLUMN(Centrality, centrality, float);
+} // namespace ptQn
+DECLARE_SOA_TABLE(MultPtQn, "AOD", "pT_Qn", ptQn::Q1, ptQn::Q2, ptQn::Q3, ptQn::Q4, ptQn::N_ch, ptQn::Centrality);
```
[the remark this answers is not in the document]
Comment (SwatiSaha-1997) [reply PWGCF/EbyEFluctuations/Tasks/TaskPtCorrelation.cxx]: Comments are added.

File: PWGCF/EbyEFluctuations/Tasks/TaskPtCorrelation.cxx [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+} // namespace o2::aod
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+struct TaskPtCorrelation_QA {
+
```
[the remark this answers is not in the document]
Comment (SwatiSaha-1997) [reply PWGCF/EbyEFluctuations/Tasks/TaskPtCorrelation.cxx]: structure is renamed according to your suggestion

File: PWGCF/EbyEFluctuations/Tasks/TaskPtCorrelation.cxx [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+      }
+    }
+    mult_ptQn(q1, q2, q3, q4, n_ch, cent);
+    // MeanPt
+    if (N > 0.0f)
+      histos.fill(HIST("hMeanPt"), cent, pT_sum / N);
+  }
+};
+
+struct TaskPtCorrelation_analysis {
```
[the remark this answers is not in the document]
Comment (SwatiSaha-1997) [reply PWGCF/EbyEFluctuations/Tasks/TaskPtCorrelation.cxx]: Done.

**Comment by SwatiSaha-1997** [comment]
The code as well as the workflows are renamed, the structure is renamed and comments are added for the columns in table

**Comment by SwatiSaha-1997** [comment]
> Please, implement my comments regarding renaming and commenting the columns/table
> 
> For the other just consider it if you want to simplify your code

This comments are updated. Please have a look at the latest changes.

**Review by SwatiSaha-1997 — COMMENTED** [review]
Requested changes are addressed

**Review by victor-gonzalez — APPROVED** [review]
Thanks for considering my comments

**Comment by SwatiSaha-1997** [comment]
> Thanks for considering my comments

Thank you for the review. Since all checks are passed, I would request you to merge this PR. Thanks in advance again,

**Pull request merged**